src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -293,6 +293,12 @@
     <PackageReference Include="WindowsAzure.Storage" Version="9.3.1" />
     <PackageReference Include="YamlDotNet" Version="6.0.0" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
```

Comment:
Can we add a comment so we can track these as a transitive dependency

Reply:
added!